File: compliance-logging-common/src/main/java/com/slalom/logging/compliance/common/MaskType.java
package com.slalom.logging.compliance.common;

import org.slf4j.Marker;
import org.slf4j.MarkerFactory;

public class MaskType {
    public static String JSON_MARKER_NAME = "JSON-COMPLIANCE";
    public static String LOMBOK_MARKER_NAME = "LOMBOK-COMPLIANCE";
    public static Marker JSON = MarkerFactory.getMarker(JSON_MARKER_NAME);
    public static Marker LOMBOK = MarkerFactory.getMarker(LOMBOK_MARKER_NAME);
}

File: compliance-logging-common/src/main/java/com/slalom/logging/compliance/common/MaskType.kt
package com.slalom.logging.compliance.common

import org.slf4j.Marker
import org.slf4j.MarkerFactory

class MaskType {
    companion object {
        const val JSON_MARKER_NAME = "JSON-COMPLIANCE"
        const val LOMBOK_MARKER_NAME = "LOMBOK-COMPLIANCE"
        val JSON: Marker = MarkerFactory.getMarker(JSON_MARKER_NAME)
        val LOMBOK: Marker = MarkerFactory.getMarker("LOMBOK-COMPLIANCE")
    }
}
